### PR TITLE
Move serverless jobs to serverless manager file

### PR DESCRIPTION
**Pull request by MichalKalke**
**Description**

Move serverless jobs to serverless manager file

Changes proposed in this pull request:

- Move serverless jobs to serverless manager file
- Adjust path to `Dockerfile` for `python39` and `gitserver` jobs
- Add new script `kyma-serverless-integration-k3d-operator.sh` to test serverless in `Serverless-Manager` repo

**Related issue(s)**
https://github.com/kyma-project/serverless-manager/issues/278

### CODE REVIEW

**Review by dbadura — DISMISSED** [review]

File: prow/jobs/kyma/tests/serverless/serverless-integration-k3d.yaml [hunk 1]
```diff
@@ -1,4 +1,89 @@
...
+              privileged: true
+              seccompProfile:
+                type: Unconfined
+              allowPrivilegeEscalation: true
+            command:
+              - "/init.sh"
+            args:
+              - "bash"
+              - "-c"
+              - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/kyma-serverless-integration-k3d-operator.sh serverless-integration serverless-contract-tests"
```
Comment (Ressetkk) [inline prow/jobs/kyma/tests/serverless/serverless-integration-k3d.yaml:38]: Please don't refer to scripts on the test infra anymore. If you need this, set up the environment and it's scripts in the `kyma-project/serverless-manager` repository.

File: prow/jobs/kyma/tests/serverless/serverless-integration-k3d.yaml [hunk 2]
```diff
@@ -1,4 +1,89 @@
...
+                type: Unconfined
+              allowPrivilegeEscalation: true
+            command:
+              - "/init.sh"
+            args:
+              - "bash"
+              - "-c"
+              - "export APP_TEST_GITHUB_SSH_AUTH_KEY=\"${GH_AUTH_PRIVATE_KEY}\" &&
+export APP_TEST_AZURE_BASIC_AUTH_USERNAME=\"${AZURE_DEVOPS_AUTH_USERNAME}\" &&
+export APP_TEST_AZURE_BASIC_AUTH_PASSWORD=\"${AZURE_DEVOPS_AUTH_PASSWORD}\" &&
```
Comment (Ressetkk) [inline prow/jobs/kyma/tests/serverless/serverless-integration-k3d.yaml:81]: This is not safe. If you by any chance leak the exported values, the internal censor will not take them up.

**Review by Ressetkk — APPROVED** [review]
Conditionally approved. Please fix the stuff ASAP.

**Review by dbadura — APPROVED** [review]

**Review by Ressetkk — APPROVED** [review]

**Pull request merged**